signer/src/testing/storage/postgres.rs

```diff
@@ -3,7 +3,7 @@
 
 use crate::error::Error;
 use crate::storage::model;
-use crate::storage::postgres::PgStore;
+use crate::storage::pgsql::PgStore;
```

Comment:
Oh, so why name the module `pgsql` instead of keeping it the same as `postgres`? 

Well, we made these testing helper modules have names that correspond to the modules that they are helping, so we should rename this testing module or rename the `pgsql` module back to `postgres`. I would prefer we keep the old `pgsql` named `postgres`, but I don't feel strongly about it.

Reply:
It wasn't by choice, the emily build containers refused to acknowledge that postgres.rs was deleted and complained about duplicate modules after converting it into a ditectory. I could try renaming it back and see if this has resolved itself.

Reply:
Oh I think this is due to a "bug" in how we build the signer binary in docker. In docker, we pull main first and then copy over our local changes in a later step. The copy doesn't overwrite existing directories, but instead copies over new or changed files. I believe that it was done this way to speed up docker builds of the signer but it has this drawback, and I've sometimes needed to fight with this bug myself.

So it looks like it's finally time to fix that bug on main.

Reply:
This should fix it https://github.com/stacks-sbtc/sbtc/pull/1704